packages/ra-core/src/controller/useEditController.spec.tsx

```diff
@@ -47,17 +47,6 @@ describe('useEditController', () => {
         expect(getByText('hello')).toBeDefined();
     });
 
-    it('should reset the redux form', () => {
```

Comment:
the test used to test the implementation, but the feature 'should reset the ~~redux~~ form' still needs to be tested, don't you think?

Reply:
We have e2e tests for it